includes/implicitflow/step3.md

```diff
@@ -0,0 +1,121 @@
+      </tr>
+      <tr>
+        <td>tenant_id</td>
+        <td>string</td>
+        <td>The ID of the specific Blackbaud customer (tenant) whose data can be accessed using the access token.  When the user grants permission to your application, they do so in the context of their organization.  The tokens we issue can only be used to access that customer's data.  We provide this value to you for informational purposes only - it is not used when calling the {{ stache.config.api_type_name }}. You may store this value in your application along with the user's access token.</td>
+      </tr>
+      <tr>
+        <td>tenant_name</td>
+        <td>string</td>
+        <td>The name of the specific Blackbaud customer (tenant) whose data can be accessed using the access token.  When the user grants permission to your application, they do so in the context of their organization.  The tokens we issue can only be used to access that customer's data.  We provide this value to you for informational purposes only - it is not used when calling the {{ stache.config.api_type_name }}. You may store this value in your application along with the user's access token.</td>
+      </tr>
+      <tr>
+        <td>legal_entity_id</td>
+        <td>string</td>
+        <td>The legal entity identifier.  A legal entity represents a Blackbaud client (organization or individual) who can try out or purchase the products, services, and solutions Blackbaud provides.  We provide this value to you for informational purposes only - it is not used when calling the {{ stache.config.api_type_name }}. You may store this value in your application along with the user's access token.</td>
+      </tr>
+      <tr>
+        <td>legal_entity_name</td>
+        <td>string</td>
+        <td>The name of the legal entity.</td>
+      </tr>
+      <tr>
+        <td>environment_id</td>
+        <td>string</td>
+        <td>The environment identifier.  An environment is a logical group of products and service instances that a legal entity is entitled to.  We provide this value to you for informational purposes only - it is not used when calling the {{ stache.config.api_type_name }}. You may store this value in your application along with the user's access token.</td>
+      </tr>
+      <tr>
+        <td>environment_name</td>
+        <td>string</td>
+        <td>The name of the environment.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<div class="alert alert-info" role="alert"><strong><em>Reminder:</em></strong> The Implicit Grant Flow provides a short-lived access token that expires in {{ stache.config.access_token_expiration_minutes }} minutes. A refresh token is not provided, so when the token expires your application should re-authorize the user again.</div>
+
+Successful redirect response example (with extra line breaks for display purposes only):
+
+<pre><code class="language-http">https://www.example.com/oauth2/callback#
```

Comment:
I did a copy/paste here from old content. However, I certainly could convert it. Will look into that!